dream-httpaf.opam

```diff
@@ -17,7 +17,7 @@ depends: [
   "dune" {>= "2.7.0"}  # --instrument-with.
   "gluten"
   "gluten-lwt-unix"
-  "h2" {< "0.13.0"}
+  "h2" {>= "0.9.0" & < "0.14.0"}
```

Comment:
Maybe remove the upper bound as we so far don't have a reason for that? WDYT?

Reply:
As the version is still <1 there can be breaking changes in any future release. Something to keep in mind.

Reply:
Sure! In the past I've set upper version constraints on as-yet unreleased versions of dependencies that were <1. To me that helped me not get distracted by new breaking releases. However, it also ensured I didn't test against new releases until much much later when I remembered why I put the upper bound.

In the end I think it's *usually* easier to put an upper bound when needed than remembering to test new releases when there's a speculative upper bound. YMMV.

Reply:
I think this MR is a good example on why it’s a good idea to have a strict upper bound: 0.13 does introduce a breaking change. :sweat_smile: